README.md

```diff
@@ -1,21 +1,42 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
 ## Use the client-side tool
 
-To access scalar-admin integrated applications, you can use the fat jar `scalar-admin-<version>-all.jar` found in [releases](https://github.com/scalar-labs/scalar-admin/releases) as follows.
+To access scalar-admin integrated applications, you can use the fat jar `scalar-admin-<version>-all.jar` found in [releases](https://github.com/scalar-labs/scalar-admin/releases) as follows
+
 ```console
-$ java -jar scalar-admin-<version>-all.jar
+$ java -jar scalar-admin-<version>-all.jar -c <Command> -s <SRV_Service_URL>
 ```
 
-You can also build the tool as follows.
+You can use the scalar-admin docker container as follows
 
 ```console
-$ ./gradlew installDist
-$ build/install/scalar-admin/bin/scalar-admin
+$ docker run -it --rm ghcr.io/scalar-labs/scalar-admin:<version> -c <Command> -s <SRV_Service_URL>
 ```
 
+You can use the scalar-admin docker container on kubernetes as follows
+
+```console
+$ kubectl run --image=ghcr.io/scalar-labs/scalar-admin:<version> --restart=Never --rm  -i scalarAdmin -- -c <Command> -s <SRV_Service_URL>
+```
+
+Note:- Commands are `PAUSE`, `UNPAUSE` and `STATS`.
+
+## Containerize the client-side tool
+
+You can create a build to containerize the scalar-admin tool as follows.
```

Comment:
English is pretty weird.
Do you mean this?
```suggestion
If you want to create a docker container, you can do it as follows.
```

Reply:
Sorry for the inconvenience.
Document updated with the above suggestion.